### PR TITLE
ActivityDescriptionActivity layout enhancement & patch some memory leaks

### PR DESCRIPTION
**Pull request by jlstoupy98**
I reworked the layout of ActivityDescriptionActivity to make it more presentable.
finish() have been added to avoid overlapping activities each time a startActivity() is called. This way, the activities don't overlap, solving some problems concerning the behavior of the back button

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -97,6 +97,7 @@ dependencies {
         exclude module: 'recyclerview-v7'
     }
     annotationProcessor 'androidx.room:room-compiler:2.2.6'
+    implementation 'de.hdodenhof:circleimageview:3.1.0'
```
Comment (Antho1426) [inline app/build.gradle:100]: What is this new implementation for ?

**Review by Antho1426 — APPROVED** [review]

**Review by bitshrine — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (jlstoupy98) [reply app/build.gradle:100]: it simply adds a tag in the xml that allows to have a circular image. I use it to enhance the design of our layout.

File: app/src/main/java/com/sdp/movemeet/map/MainMapFragment.java [hunk 2]
```diff
@@ -54,7 +50,7 @@
     //current location of the user on the map
     private Location currentLocation;
 
-    private FirebaseAuth fAuth = FirebaseAuth.getInstance();
+    private final FirebaseAuth fAuth = FirebaseAuth.getInstance();
```
Comment (RoxaneBurri) [inline app/src/main/java/com/sdp/movemeet/map/MainMapFragment.java:53]: We have to put every fAuth in final

Comment (Antho1426) [reply app/src/main/java/com/sdp/movemeet/map/MainMapFragment.java:53]: Why is this necessary?

**Review by Antho1426 — APPROVED** [review]

**Pull request merged**